etmain/animations/human/base/body.aninc

```diff
@@ -22,22 +22,14 @@
 		//	|   				|   	|   |   |   	|   |   |   <priority>
 		//	|   				|   	|   |   |   	|   |   |   |
 
-		// DEATHS {{{ - do not change the order!
```

Comment:
btw. I've retained the `do not change the order!` warning so far, but after checking a couple of times and thinking about it - I see no reason anymore why the order would matter, especially with the priority system now.

Any check that checks for specific death animations is brittle to begin with and if it has to be should rely on either `name` or `nameHash`, instead of an animation index number that can easily change.

Reply:
I've finally figured out why this warning was there and put it in with a proper explanation of what it is about.

The first frames of the the first animation defined of the `body.aninc` file is used for a single or two frames when a player spawns and then lerps to the animation we actually want.

I've tried a bunch of different parts in the body animation and ended up with one that causes the least jarring no matter how many frames are taken off it and no matter what weapon a player spawns with as best as possible.

Should be significantly better than the previous spawn animation (which was literally the first death animation).